README.md

```diff
@@ -64,12 +64,25 @@ Then you should be able to run `expo start` to access a local build of the app o
 
 Note: To test the app with dummy data, you will have to run a local instance of the backend in another terminal window. Backend repo/instructions: https://github.com/HotSoupRepos/HotSoup-backend
 
-In the MapList screen component, there is a hardcoded IP address, you will have to change it to your own computer's IPv4 address to make requests to the running backend. You CANNOT use localhost, since the app is running on your phone/simulator.
+You can find the api request url at src/service/soupKitchenServce.ts. There is a hardcoded url including a unique IP address stored in a variable. You will have to change it to your own computer's IPv4 address to make requests to the running backend. You CANNOT use localhost, since the app is running on your phone/simulator.
```

Comment:
Typo here in the spelling of service I think?

Reply:
woops! will fix

Reply:
is this resolved? if so, let's mark the conversation as resolved